### PR TITLE
Handle invalid dates for dashboard search

### PR DESCRIPTION
**Pull request by dideler**
A flash warning will now be shown if an invalid date is entered.

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -7,26 +7,29 @@ def index
     @from_date = params[:from]
     @to_date = params[:to]
 
-    redirect_to root_path(from: Time.zone.today, to: Time.zone.today) if @query.blank? &&
-                                                                         @from_date.blank? &&
-                                                                         @to_date.blank?
+    search_for_tickets_deployed_today if empty_search?
```
Comment (0mega) [inline app/controllers/home_controller.rb]: nice refactor

File: app/views/home/index.html.haml [hunk 2]
```diff
@@ -10,30 +10,28 @@
       %input#to_date.input-sm.form-control{:name => "to", :type => "text", :style => "width: 120px; height: 34px;", :value => @to_date}/
   %input.btn.btn-primary{:type => "submit", :value => "Search"}/
 
-- if @query || @from_date || @to_date
```
Comment (dideler) [inline app/views/home/index.html.haml]: Removed this because now a date will always be set, so the condition didn't matter.

**Comment by 0mega** [comment]
LGTM

File: app/controllers/home_controller.rb (hunk 1 above)
Comment (andistuder) [inline app/controllers/home_controller.rb]: No need to change, but for future reference: I'd prefer if the private method name is explicit the method it calls on ActionController.

**Pull request merged**